pylabrobot/resources/trough.py

```diff
@@ -13,9 +21,18 @@ def __init__(
     size_y: float,
     size_z: float,
     max_volume: float,
+    material_z_thickness: float = 0,
+    true_dz: float = 0,
     category: Optional[str] = "trough",
-    model: Optional[str] = None
+    model: Optional[str] = None,
+    bottom_type: Union[TroughBottomType, str] = TroughBottomType.UNKNOWN,
+    compute_volume_from_height: Optional[Callable[[float], float]] = None,
+    compute_height_from_volume: Optional[Callable[[float], float]] = None,
```

Comment:
what do you think about moving these to `Container` after the PR?

Reply:
Love it, let's do that!

Reply:
Done with https://github.com/PyLabRobot/pylabrobot/commit/f9d496cedc83fbd065e9d3c2b045e0038522feaa

Reply:
done with https://github.com/PyLabRobot/pylabrobot/commit/60cc8eddc359f7cb7457e40e520561dc8cac2a86